basic-app.Rmd

```diff
@@ -255,7 +255,7 @@ server <- function(input, output, session) {
 }
 ```
 
-1.  Extend the app from the previous exercise to allow the user to set the value of the 
+2.  Extend the app from the previous exercise to allow the user to set the value of the 
```

Comment:
Can you leave these as 1 please? If they’re not getting sequentially numbered in the output, it’s a sign that something else is wrong.

Reply:
Ah, got it!